sdk/ai/Azure.AI.Agents.Persistent/CHANGELOG.md

```diff
@@ -1,17 +1,12 @@
 # Release History
 
-## 1.1.0-beta.3 (Unreleased)
+## 1.1.0-beta.3 (2025-06-27)
 
 ### Features Added
 
 - Tracing for Agents. More information [here](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/ai/Azure.AI.Agents.Persistent/README.md#tracing).
 - Convenience constructor for BingCustomSearchToolParameters
-
-### Breaking Changes
-
-### Bugs Fixed
-
-### Other Changes
+- Automatically function toolcalls.  More information [here](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/ai/Azure.AI.Agents.Persistent/README.md#function-call-executed-automatically).
```

Comment:
Automatic execution of function tool calls. More information [here](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/ai/Azure.AI.Agents.Persistent/README.md#function-call-executed-automatically).

Reply:
https://github.com/Azure/azure-sdk-for-net/pull/50945